content/guides/authors/data.json

```diff
@@ -0,0 +1,20 @@
+{
+  "rishi-raj-jain": {
+    "name": "Rishi Raj Jain",
+    "position": "Developer",
+    "bio": "Previously, Technical Support Engineer @lmsqueezy, Technical CSM @Edgio & Ambassador @storyblok.",
```

Comment:
May we use just "Software Engineer" instead?

Reply:
Or how about "Technical Writer"?

Reply:
@rishi-raj-jain sure, this is just mock data, I found about you. Please feel free to send me your real data (name, position, bio, and photo) if you like to change something.

Reply:
All's fine, how about using "Technical Writer" instead of the big description?

Reply:
@rishi-raj-jain ok sure, please check the update

Reply:
Looks good :D